### PR TITLE
feat: remake zk stack quickstart to use new zk cli

### PR DESCRIPTION
**Pull request by zk-Lumi**
# Description

Updated ZK Stack quickstart guide for new ZK CLI

NOTE: Blockexplorer + dapp-portal parts need to be updated still - @Deniallugo will work with devrel to do after this PR

### CODE REVIEW

File: content/10.zk-stack/20.running-a-zk-chain/10.locally.md [hunk 1]
```diff
@@ -16,60 +16,127 @@ to set up dependencies on your machine (don't worry about the Environment sectio
...
-    export PATH=$ZKSYNC_HOME/bin:$PATH
+    zk_inception ecosystem create
     ```
 
-1. Build latest version of zk tools by just running `zk` on the root of the project.
+    - Give the ecosystem a name, for example "ZKsync Elastic Chain".
+
+    - For the origin of zksync-era repository, select "Clone for me".
+
+    - For the L1 network select "Localhost" to use the local geth docker container.
```
Comment (Deniallugo) [inline content/10.zk-stack/20.running-a-zk-chain/10.locally.md:53]: reth, not geth

File: content/10.zk-stack/20.running-a-zk-chain/10.locally.md [hunk 2]
```diff
@@ -16,60 +16,127 @@ to set up dependencies on your machine (don't worry about the Environment sectio
...
-1. Last, start the wizard and follow instructions to set up and deploy your new ZK chain by running `zk stack init`
+3. If the ecosystem has never been deployed before, initialization is required:
 
-    - Initially you want to `Configure new chain`
+    ```bash
+    zk_inception ecosystem init
+    ```
 
-    - Give it a name and chain id.
+    - Follow the wizard to choose options for your deployment. For default options, you can simply click enter for all prompts.
```
Comment (Deniallugo) [inline content/10.zk-stack/20.running-a-zk-chain/10.locally.md:83]: or you can use `--dev` for using all defaults

File: content/10.zk-stack/20.running-a-zk-chain/10.locally.md [hunk 3]
```diff
@@ -16,60 +16,127 @@ to set up dependencies on your machine (don't worry about the Environment sectio
...
-    zk server --components "http_api,eth,state_keeper,housekeeper"
+    zk_inception server
     ```
 
-1. You can now run transactions and start playing with your ZK chain by using the RPC available at <http://localhost:3050>.
+    If you setup multiple ZK chains, you can specify the chain you are running by providing `--chain <chain_name>` argument.
+
+### Your ZK chain is now deployed & running
+
+Your Elastic Chain ecosystem and your ZK chain(s) are now deployed to the base chain (most likely a local geth docker
```
Comment (Deniallugo) [inline content/10.zk-stack/20.running-a-zk-chain/10.locally.md:126]: reth

**Review by Deniallugo — COMMENTED** [review]
almost all good

**Comment by zk-Lumi** [comment]
@itsacoyote mind merging?

File: content/10.zk-stack/20.running-a-zk-chain/10.locally.md [hunk 4]
```diff
@@ -16,74 +16,141 @@ to set up dependencies on your machine (don't worry about the Environment sectio
 
 ## Deploying locally
 
-1. Clone the zksync-era repo (or pull the latest if you've already cloned it) and go to the root of it:
+::callout{icon="i-heroicons-exclamation-triangle" color="amber"}
+This guide is for setting up an Elastic Chain ecosystem and for spinning up your own ZK chains
+using ZK Stack. If you just want play around with a pre-built ZKsync chain, you can use `zksync-cli dev` to run docker containers.
+Learn more [here](/build/tooling/zksync-cli).
```
Comment (itsacoyote) [inline content/10.zk-stack/20.running-a-zk-chain/10.locally.md:22]:
```suggestion
Learn more on [zksync-cli](/build/tooling/zksync-cli).
```

**Review by itsacoyote — APPROVED** [review]

**Pull request merged**